components/network/tor/tor.p5m

```diff
@@ -31,15 +33,20 @@ dir path=var/lib/tor group=daemon mode=0755 owner=daemon
 dir path=var/log/tor group=daemon mode=0755 owner=daemon
 
 file etc/tor/torrc.sample path=etc/tor/torrc mode=0644 preserve=true
-
-file usr/bin/tor path=usr/lib/tor mode=0555
+file usr/bin/tor path=usr/bin/tor mode=0555
```

Comment:
This should be now enough:
```suggestion
file usr/bin/tor
```

Reply:
Agree and fixed.

Reply:
I am sorry, I mislead you here, it should be: `file path=usr/bin/tor`. Otherwise it won't publish.

Reply:
I fixed it in this PR.